FeedStoreChallenge/ManagedCache.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ManagedCache+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by Andreas Link on 13.06.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
+	@NSManaged internal var timestamp: Date
+	@NSManaged internal var feed: NSOrderedSet
```

Comment:
The convention in this codebase is to omit the default access level keyword of `internal`, please remove it.
```suggestion
	@NSManaged var timestamp: Date
	@NSManaged var feed: NSOrderedSet
```